src/utils/handleResponse.js

```diff
@@ -1,18 +1,21 @@
 import newError from './errors';
 
 const handleResponse = (response) => {
+  if (!response.ok) {
```

Comment:
Ah good catch, I didn't think of that use case. 

Is the current implementation inside the if-statement correct though? My first thought is that it should return a resolved promise, rather than ``null``?